### PR TITLE
Issue-28: Add Artillery Load Test tool

### PR DESCRIPTION
**Pull request by tombui99**
- Added Artillery tool in Docker's packages and display the results in the terminal
- Added "--artilllery" as an option in command line
- Added functions to the API to upload Artillery result json file to Azure Blob Storage

### CODE REVIEW

File: docker/index.js [hunk 1]
```diff
@@ -80,6 +82,11 @@ const _getAgrs = () => {
 			describe: 'Include Lighthouse audit',
 			type: 'boolean',
 			default: false,
+		})
+		.option('artillery', {
+			describe: 'Include Artillery test',
+			type: 'boolean',
+			default: false,
```
Comment (nvhoanganh) [inline docker/index.js:89]: can we make this default = true

File: docker/utils.js [hunk 2]
```diff
@@ -244,6 +244,38 @@ exports.readLighthouseReport = (folder, writeLog) => {
...
+	);
+
+	const atrSummary = {
+		timestamp: atr.aggregate.timestamp,
+		scenariosCreated: atr.aggregate.scenariosCreated,
+		scenariosCompleted: atr.aggregate.scenariosCompleted,
+		requestsCompleted: atr.aggregate.requestsCompleted,
+		rpsCount: atr.aggregate.rps.count,
+		latencyMedian: atr.aggregate.latency.median,
+		scenarioCount: JSON.stringify(atr.aggregate.scenarioCounts),
```
Comment (nvhoanganh) [inline docker/utils.js:273]: why this?

**Review by nvhoanganh — COMMENTED** [review]
looks really good, well done

File: docker/index.js (hunk 1 above)
Comment (tombui99) [reply docker/index.js]: Done - default set to true

File: docker/utils.js (hunk 2 above)
Comment (tombui99) [reply docker/utils.js:273]: This is the summary based on the aggregate result from Artillery

Comment (nvhoanganh) [reply docker/utils.js:273]: I mean why we have to JSON.strigify it?

**Review by nvhoanganh — APPROVED** [review]
LGTM

**Pull request merged**